deploy/hco.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubevirt-hyperconverged
+
+---
+apiVersion: operators.coreos.com/v1
+kind: CatalogSourceConfig
+metadata:
+  name: hco-catalogsource-config
+  namespace: kubevirt-hyperconverged
+spec:
+  targetNamespace: kubevirt-hyperconverged
+  packages: hco-operatorhub
+  csDisplayName: Community Operators
+  csPublisher: Community
+
+---
+apiVersion: operators.coreos.com/v1
+kind: CatalogSourceConfig
+metadata:
+  name: installed-community-kubevirt-hyperconverged
+  namespace: openshift-marketplace
```

Comment:
why is it that you have two CatalogSourceConfigs, one in the HCO namespace and the other in marketplace? I owuld be surprised if both were needed.

Reply:
Maybe one is supposed to be an operatorSource?

Reply:
not sure how it got in there since they're identical resources with different names.  Removing it.

The CommunityOperators operatorsource already exists on 4.2 openshift deployments by default so we don't need to create it.